README.md

```diff
@@ -57,11 +57,8 @@ If adding a camera which does not needed mapped because the focus point is given
 
 Known Issues
 --------
-1. Not currently working on Windows. 
```

Comment:
What is it about mirroring that needs 865 assets changed? Did your script modify all the files which is why it's showing as changed? When the PR has this many files, Github doesn't render it very well. It makes it hard on me.